### DISCUSSION TITLE
Getting the version (not tagName) in Mustache template

### DISCUSSION THREAD

**Discussion by triceo** [jreleaser/jreleaser#1422] [Q&A] [answered]
Hello there,
I wish to render a code like this in my changelong:

    ```xml
      <dependency>
        <groupId>a</groupId>
        <artifactId>b</artifactId>
        <version>1.0.0</version>
      </dependency>
    ```

Unfortunately, if I use the {{tagName}} placeholder, I will get `v1.0.0`, not `1.0.0`.
Is there a property somewhere which allows me to access the version number, not the tag name?

**Comment by aalmiray** [comment, marked as answer]
Yes, you may use any of the following keys:

```
        versionMajor: 1
        versionMinor: 0
        versionPatch: 0
        versionNumber: 1.0.0
        versionWithUnderscores: 1_0_0
        versionWithDashes: 1-0-0
        versionNumberWithUnderscores: 1_0_0
        versionNumberWithDashes: 1-0-0
        effectiveVersionWithUnderscores: 1_0_0
        effectiveVersionWithDashes: 1-0-0
```

**Comment by triceo** [comment]
@aalmiray I tried to apply the `versionNumber` key in this snippet:

    ```xml
    <dependency>
      <groupId>a</groupId>
      <artifactId>b</artifactId>
      <version>{{versionNumber}}</version>
    </dependency>
    ```

The end result was an empty string instead.
Something I'm missing? Do I need to set the number somewhere, perhaps?

**Reply by aalmiray** [reply]
Hmm 'versionNumber` should had worked. Perhaps it requires a prefix in which case `projectVersionNumber` should do the trick. https://jreleaser.org/guide/latest/reference/name-templates.html#_project

**Comment by triceo** [comment]
In the end, `projectVersion` was the one that worked.
